subsys/bluetooth/controller/Kconfig

```diff
@@ -406,7 +406,7 @@ config BT_CTLR_PHY_2M
 
 config BT_CTLR_PHY_CODED
 	bool "Coded PHY Support"
-	depends on (BT_CTLR_PHY || BT_CTLR_ADV_EXT) && SOC_NRF52840
+	depends on (BT_CTLR_PHY || BT_CTLR_ADV_EXT) && HAS_HW_NRF_RADIO_BLE_CODED
```

Comment:
Isn't the line too long now?

Reply:
Yes, but this is common in Kconfig files, AFAIK